src/util.ts

```diff
@@ -954,3 +954,14 @@ export function runGodoc(cwd: string, packagePath: string, receiver: string, sym
 		});
 	});
 }
+
+export function getTimeoutConfiguration(operationType: string, execFileOptions: {[key: string]: any}) {
+	let editor = vscode.window.activeTextEditor;
```

Comment:
Not all features of this extension need the active editor. For example, any of the build/lint/vet/test commands for workspace or installing the Go tools. So, instead of using the active editor here, I would suggest passing the `goConfig` as a parameter to this function. Almost all files access the configuration, so it wouldnt be too much of a problem to pass it along.